src/sensai/featuregen.py

```diff
@@ -243,21 +243,30 @@ class FeatureGeneratorFlattenColumns(RuleBasedFeatureGenerator):
     Instances of this class take columns with vectors and creates a dataframe with columns containing entries of
     these vectors.
 
-    For example, if columns "vec1", "vec2" contain vectors of dimensions dim1, dim2, a datafrane dim1+dim2 new columns
+    For example, if columns "vec1", "vec2" contain vectors of dimensions dim1, dim2, a dataframe with dim1+dim2 new columns
```

Comment:
If we're correcting this, let's go all the way: "data frame"

Reply:
Done